reference/configuration/framework.rst

```diff
@@ -70,18 +70,18 @@ Configuration
     * `gc_maxlifetime`_
     * `save_path`_
 * `templating`_
-    * `assets_version`_
-    * `assets_version_format`_
     * `hinclude_default_template`_
     * :ref:`form <reference-templating-form>`
         * `resources`_
-    * `assets_base_urls`_
```

Comment:
It'd be nice to make sure these `#anchor` links still work (e.g. http://symfony.com/doc/current/reference/configuration/framework.html#assets-version). You can do this by adding a reference above the new, equivalent section:

```
.. _assets-version:

version
-------
```